crypto.go

```diff
@@ -0,0 +1,96 @@
+package faker
+
+import (
+	"strings"
+)
+
+// Crypto is a faker struct for generating bitcoin data
+type Crypto struct {
+	Faker *Faker
+}
+
+var (
+	bitcoinMin = 26
+	bitcoinMax = 35
+)
+
+// Checks whether the ascii value provided is in the exclusion for bitcoin.
+func inExclusion(check int) bool {
+	// switch probably seems to be the fastest as compared to list/map
+	// https://stackoverflow.com/questions/15323767/does-go-have-if-x-in-construct-similar-to-python
+	switch check {
+	// Ascii for uppercase letter "O", uppercase letter "I", lowercase letter "l", and the number "0"
+	case
+		48,
+		73,
+		79,
+		108:
+		return true
+	}
+	return false
+}
+
+// Decide whether to get digit, uppercase, or lowercase. returns the ascii range to do IntBetween on
+func getBitcoinRange(f *Faker) (int, int) {
+	dec := f.IntBetween(0, 2)
+	if dec == 0 {
+		return 48, 57
+	} else if dec == 1 {
+		return 65, 90
+	} else {
+		return 97, 122
+	}
+}
+
+// Helper function to return a bitcoin
```

Comment:
just to confirm, by `return a bitcoin` you mean one bitcoin hash? if so, can you change this message to be clear that it is a bitcoin hash?